kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/marshalling/JSONMarshallerTest.java

```diff
@@ -317,4 +319,188 @@ public void testLegalizeWrapObjectFieldnames() throws Exception {
+        assertEquals(java.time.LocalTime.class, convertedSupportedDateType.get("localtime").getClass());
+        assertEquals(java.time.OffsetDateTime.class, convertedSupportedDateType.get("offsetDateTime").getClass());
+        assertEquals(java.util.Date.class, convertedSupportedDateType.get("utildate").getClass());
+
+        assertEquals(SupportedlDate.class, convertedSupportedDateType.get("sqldate").getClass());
+
+        SupportedlDate supportedDate = (SupportedlDate) convertedSupportedDateType.get("sqldate");
+        assertEquals(java.util.Date.class, supportedDate.getUtildate().getClass());
+        assertEquals(java.time.LocalDateTime.class, supportedDate.getLocalDateTime().getClass());
+        assertEquals(java.time.LocalDate.class, supportedDate.getLocalDate().getClass());
+        assertEquals(java.time.LocalTime.class, supportedDate.getLocalTime().getClass());
+        assertEquals(java.time.OffsetDateTime.class, supportedDate.getOffsetDateTime().getClass());
+    }
+}
+
+class SupportedlDate {
+
+    private java.util.Date utildate;
+    private java.time.LocalDate localDate;
+    private java.time.LocalDateTime localDateTime;
+    private java.time.LocalTime localTime;
+    private java.time.OffsetDateTime offsetDateTime;
+
+    public SupportedlDate() {
+    }
+
+    public java.util.Date getUtildate() {
+        return this.utildate;
+    }
+
+    public void setUtildate(java.util.Date utildate) {
+        this.utildate = utildate;
+    }
+
+    public java.time.LocalDate getLocalDate() {
+        return this.localDate;
+    }
+
+    public void setLocalDate(java.time.LocalDate localDate) {
+        this.localDate = localDate;
+    }
+
+    public java.time.LocalDateTime getLocalDateTime() {
+        return this.localDateTime;
+    }
+
+    public void setLocalDateTime(java.time.LocalDateTime localDateTime) {
+        this.localDateTime = localDateTime;
+    }
+
+    public java.time.LocalTime getLocalTime() {
+        return this.localTime;
+    }
+
+    public void setLocalTime(java.time.LocalTime localTime) {
+        this.localTime = localTime;
+    }
+
+    public java.time.OffsetDateTime getOffsetDateTime() {
+        return this.offsetDateTime;
+    }
+
+    public void setOffsetDateTime(java.time.OffsetDateTime offsetDateTime) {
+        this.offsetDateTime = offsetDateTime;
+    }
+}
+
+class Person {
+
+    private java.lang.String fullname;
+    private java.lang.Integer age;
+    private java.time.LocalDateTime dateBirth;
+    private java.util.Date date;
+    private java.sql.Date sqlDate;
+    private java.sql.Timestamp timestamp;
+    private java.time.LocalTime localTime;
+    private java.time.OffsetDateTime offsetDateTime;
+
+    public java.sql.Date getSqlDate() {
+        return sqlDate;
+    }
+
+    public void setSqlDate(java.sql.Date sqlDate) {
+        this.sqlDate = sqlDate;
+    }
+
+    public Timestamp getTimestamp() {
```

Comment:
this one is imported as package, yet in other occurrences its FQN is used, java.sql.Timestamp, may be import by accident?